custom-vu/src/main/kotlin/jces1209/vu/action/LogInWithAtlassianId.kt

```diff
@@ -38,6 +39,7 @@ class LogInWithAtlassianId(
 
     private fun logIn() {
         jira.goToLogin()
+        Thread.sleep(Random().nextInt(Duration.ofMinutes(5).toMillis().toInt()).plus(Duration.ofMinutes(5).toMillis()))
```

Comment:
Why is it necessary to add a sleep?

Reply:
We faced the Auth0 rate-limiting problem for our tenants.
In our case, we run JPT via bamboo jobs which might take the same IP, and we use the same single user in JPT for logins, so as per the Auth0 limit if you have the same user with the same IP logging more than 20times in a 1min then it will block that users hence our VU logins were failing, to extend the duration of the login we added random wait between each login.
- I have flag out this with property key which we pass for our runs if that value presents then only it will add the waits otherwise it will run as normal.

Reply:
@dmewada-atlas does it make sense to add an ENV var to turn it off by default? Most people run JPT in an env without facing this issue, but only via the Jumpbox, so this could slow down the test for everyone, which is not ideal.

Reply:
oh sorry, I read an outdated code, see you wrap it in an `if` in the new code. Please ignore me